staging/src/k8s.io/apimachinery/pkg/api/testing/codec.go

```diff
@@ -14,7 +14,7 @@ See the License for the specific language governing permissions and
 limitations under the License.
 */
 
-package testing
+package apitesting
```

Comment:
why is the package name changed?

Reply:
> the inconsistency between dir name and package name makes me confused

moved the package. Calling your package `testing` is guaranteed to conflict in *every* test file since they all import `testing`.  This just fixes the problem that all callers had to work around.